spark/src/main/scala/org/apache/spark/sql/delta/commands/CreateDeltaTableCommand.scala

```diff
@@ -313,7 +316,7 @@ case class CreateDeltaTableCommand(
         protocol.foreach { protocol =>
           txn.updateProtocol(protocol)
         }
-        Nil
+        Nil ++ ClusteredTableUtils.getDomainMetadataOptional(table, txn)
```

Comment:
`ClusteredTableUtils.getDomainMetadataOptional(table, txn).toSeq ` is cleaner?

Reply:
thanks for the review, updated to use this.